product-service/handlers/get-products-by-id.ts

```diff
@@ -0,0 +1,22 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import { ProductsService } from '../services/products-service';
+
+
+export const getProductsById: APIGatewayProxyHandler = async (_event, _context) => {
```

Comment:
is this endpoint works as expected since i don't see according headers as for get-product-list?

Reply:
Yep, headers are not added here since it's not used on BE, so we don't make calls from different origin here